### ISSUE TITLE
For Cycle 15

### ISSUE THREAD

**Issue by bounhun** [bisq-network/compensation#626]
## Summary

> _Specify the total amount of BSQ you are requesting, along with the USD total and BSQ/USD rate (don't include the brackets!):_

- **BSQ requested**: **`750`** 
- USD requested: 487.5 
- BSQ rate: 0.65 USD per BSQ 
- Previous compensation request (if applicable): #599  

## Contributions delivered

> _Add contributions you have [delivered](https://github.com/bisq-network/proposals/issues/19) and roles you have performed here as new rows in the table below. Role line-items should include an asterisk (*) in the team column._

| Title | Team | USD | Link | Notes |
| --- | --- | --- | --- | --- |
| ETH bounty | Growth | 325 | https://github.com/bisq-network/growth/issues/177#issuecomment-661022501 |  |
|LTC bounty|Growth|162.5| https://github.com/bisq-network/growth/issues/177#issuecomment-661022501 ||

## Contributions in progress

> _Provide links to work you're involved with that is still [in progress](https://github.com/bisq-network/proposals/issues/19). **This section is optional (the linter ignores it),** and is for your own benefit in keeping track of what you're doing and keeping other contributors up to date with the same._

**Comment by m52go** [comment]
[Approved](https://bisq.wiki/Compensation#Review_team_member_requests), let's give it another go. It's been a good effort, with good results lately.

As noted elsewhere, we should make thresholds for future bounties to enable partial claims.

**Comment by bounhun** [comment]
I have linked twice the ETH bounty because there's where I exposed my reasons to try again. I'll expose them again too, I'm open for discussion and I want to know your opinion:

- Spent about 100USD in fees.
- I've been 5 months placing orders almost 24/7 and I plan to continue doing it, although maybe less hours a day.
- Although trade volume has not significantly risen, number of trades did.
- It's embarrassing to use this reason, but it's still true: I could have easily claim this bounty just placing a couple of self trades every month to meet the trading volume goal. Reducing the bounty by half, I'm saving the DAO 750 BSQ.
